lib/promises.helper.js

```diff
@@ -43,13 +43,25 @@ function handlePromise(parameterIndex, original) {
     return fn;
 }
 
-prototype = jasmine.Env.prototype;
+if (Object.keys(jasmine.Env.prototype).length === 0) {
+    version = 2;
+    prototype = jasmine.getEnv();
+} else {
+    version = 1;
+    prototype = jasmine.Env.prototype;
+}
```

Comment:
The first change ended up being unnecessary, as `currentEnv_` is present in all versions of Jasmine.

The second change: 154b26a